src/ace_header.F90

```diff
@@ -144,6 +145,10 @@ module ace_header
     integer                :: urr_inelastic
     type(UrrData), pointer :: urr_data => null()
 
+    ! Multipole data
+    logical                       :: mp_present
+    type(MultipoleArray), pointer :: multipole => null()
```

Comment:
I only did this due to the symmetry in structure with the above URR block, and the only reason the URR method uses it is to shorten some lines of code by doing `urr => nuc % urr_data`.  I changed it, but it may be worthwhile to do the same for URR.